### PR TITLE
Ratings block

### PR DESCRIPTION
**Pull request by williambsm**
Fix https://github.com/adobe/express-website-issues/issues/315

Test URL:
https://ratings-frontend--express-website--webistry-development.hlx3.page/drafts/solene/ratings?lighthouse=on

### CODE REVIEW

**Comment by rofe** [comment]
@So1ene FYI, the lottie player has now been added to the project via a [different PR](https://github.com/adobe/express-website/pull/389/commits/c0dad163e9be931f3eb613275e76a24b6d398ab8), so I think we should reuse that one now.

cc @williambsm

File: express/blocks/columns/columns.css [hunk 1]
```diff
@@ -584,7 +584,7 @@ main .fixed-button.button-container {
   left: 0;
   width: 100%;
   padding-bottom: 20px;
-  z-index: 2;
+  z-index: 4;
```
Comment (rofe) [inline express/blocks/columns/columns.css:587]: Why is this needed?

File: express/scripts/scripts.js [hunk 2]
```diff
@@ -109,6 +109,57 @@ export function getMeta(name) {
...
+
+// Lazy-load lottie player if you scroll to the block.
+export function lazyLoadLottiePlayer($block = null) {
+  const loadLottiePlayer = () => {
+    if (window['lottie-player']) return;
+
+    const script = document.createElement('script');
+    script.type = 'text/javascript';
+    script.async = true;
+    script.src = 'https://unpkg.com/@lottiefiles/lottie-player@latest/dist/lottie-player.js';
```
Comment (rofe) [inline express/scripts/scripts.js:125]:
```suggestion
    script.src = '/express/scripts/lottie-player.1.5.6.js';
```
We should now use `/express/scripts/lottie-player.1.5.6.js` instead of loading the external file.

**Review by rofe — CHANGES_REQUESTED** [review]

File: express/blocks/columns/columns.css (hunk 1 above)
Comment (So1ene) [reply express/blocks/columns/columns.css]: Oops, fixed!

**Comment by rofe** [comment]
Todo based on today's review:
- [ ] block option `totals` to display actual values (average stars and number of votes)

File: express/blocks/ratings/ratings.js [hunk 3]
```diff
@@ -0,0 +1,517 @@
...
+  // Countdown timer to auto-submit
+  const countdown = (bool) => {
+    if (bool) {
+      $timer.innerHTML = getLottie('countdown', '/express/blocks/ratings/countdown.json', false, true, false, false);
+      let counter = 10;
+      window.ratingSubmitCountdown = setInterval(() => {
+        if (counter > 0) {
+          counter -= 1;
+          // eslint-disable-next-line no-console
+          console.log(`rating will be submitted in: ${counter + 1} seconds.`);
```
Comment (rofe) [inline express/blocks/ratings/ratings.js:214]:
```suggestion
          // eslint-disable-next-line no-console
          console.log(`rating will be submitted in: ${counter + 1} seconds.`);
```
I think we should remove (or comment) this in the production version.

**Review by rofe — CHANGES_REQUESTED** [review]

**Review by rofe — APPROVED** [review]
🎉
